### PR TITLE
Add Rspec

### PR DESCRIPTION
**Pull request by sawleha16**

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ENV["RAILS_ENV"] ||= "test"
 require_relative "../config/environment"
 require "rails/test_help"
+require 'capybara/rails'
```
Comment (ahmad19) [inline test/test_helper.rb]: we don't need this file.

File: Gemfile [hunk 2]
```diff
@@ -71,4 +72,5 @@ group :test do
   gem "capybara"
   gem "selenium-webdriver"
   gem "webdrivers"
+  gem 'database_cleaner-active_record'
```
Comment (ahmad19) [inline Gemfile:77]: You need to add the configuration for this gem.

File: spec/models/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (ahmad19) [inline spec/models/user_spec.rb]: write some tests for this model. Refer manageursales repo

File: test/test_helper.rb (hunk 1 above)
Comment (sawleha16) [reply test/test_helper.rb]: okay

**Review by ahmad19 — APPROVED** [review]

**Pull request merged**